index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <main class="main-container">
+      <header class="toolbar">
+        <a>Welcome</a>
+        <span> <img src="images/toggle.png" alt="toggle icon" /> </span>
+      </header>
+      <section class="top-header">
+        <div class="header-title">
+          <h1>Hey there. I'm Bamsi</h1>
+          <h2 class="color-green">I'm a software developer</h2>
+        </div>
+        <div class="header-body">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="icon-wrapper">
+          <ul class="icons">
+            <li>
+              <a href="#"><img src="images/cat.png" alt="cat icon" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/in.png" alt="linkldn icon" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/v.png" alt="two icon" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="images/twitter.png" alt="twitter icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/m.png" alt="m icon" /></a>
+            </li>
+          </ul>
+        </div>
+
+        <div class="down-arrow">
+          <img src="images/arrow.png" alt="arrow" />
+        </div>
```

Comment:
- [ ] Please fix the position of the down arrow and make it match [Figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
- [ ] 
![image](https://user-images.githubusercontent.com/74744787/159047590-b9d978c7-a2ca-44ea-a919-2c0b35e2b2b7.png)